admin/preprint_providers/urls.py

```diff
@@ -17,4 +17,6 @@
     url(r'^(?P<preprint_provider_id>[a-z0-9]+)/delete/$', views.DeletePreprintProvider.as_view(), name='delete'),
     url(r'^(?P<preprint_provider_id>[a-z0-9]+)/export/$', views.ExportPreprintProvider.as_view(), name='export'),
     url(r'^(?P<preprint_provider_id>[a-z0-9]+)/share_source/$', views.ShareSourcePreprintProvider.as_view(), name='share_source'),
+    url(r'^whitelist_add/$', views.SharePreprintProviderWhitelistAdd.as_view(), name='whitelist_add'),
```

Comment:
You should be able to have one endpoint rather than 2 if you make these URLs more RESTful:

```python
POST .../whitelist/
DELETE .../whitelist/
```